Firestore/Source/Model/FSTFieldValue.h

```diff
@@ -81,7 +114,7 @@ typedef NS_ENUM(NSInteger, FSTTypeOrder) {
  */
 @interface FSTNullValue : FSTFieldValue
 + (instancetype)nullValue;
-- (id)value;
+- (id)valueWithOptions:(FSTFieldValueOptions *)options;
```

Comment:
Can't we make a stronger return type here? Isn't this always of type `NSNull`?

Reply:
Done